Procfile

```diff
@@ -1,2 +1,3 @@
 web:         bundle exec passenger start -p $PORT --max-pool-size $PASSENGER_MAX_POOL_SIZE
 worker:      QUEUES=default,paperclip,mailers bundle exec rake jobs:work
+image_reprocess_worker: QUEUE=image_reprocess bundle exec rake jobs:work
```

Comment:
@rap1ds A specific queue makes it easier to recognize when all images have been processed in order to minimize the time extra workers are required. Necessary? Hardly.